lib/bolt/util.rb

```diff
@@ -0,0 +1,47 @@
+module Bolt
+  module Util
+    # CODEREVIEW I hate mixing in random modules
+    class << self
```

Comment:
I think the common pattern for this is to define these as standard methods and then `extend self`. I guess all that accomplishes is the ability to mixin the module, which there's not a good reason for.

Reply:
Is there a reason not to?
Should we just mix this in anywhere we need it to be more idiomatic?

Reply:
I don’t think it makes sense to mix it in, since it’s basically semantically void. May as well just treat it like a static class.